### PR TITLE
Adds German translation

### PR DESCRIPTION
**Pull request by ellingen**

### CODE REVIEW

File: src/langs/de.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//Add strings
+gdprCookieNoticeLocales.en = {
```
Comment (dennzo) [inline src/langs/de.js:2]: This should be gdprCookieNoticeLocales.de

Then in the gulpfile all languages should be loaded, so you can access them.
Currently only en is loaded.

**Comment by gratzl-dev** [comment]
You are overriding the english localization....

**Comment by striezel** [comment]
> You are overriding the english localization....

Exactly.
Since there has been no further reply to this issue for half a year, I took the liberty to fix this issue and created another PR with that translation, but without the overriding part: #34

File: src/langs/de.js (hunk 1 above)
Comment (striezel) [reply src/langs/de.js:2]: > This should be gdprCookieNoticeLocales.de

Correct. Fixed in #34.

**Comment by striezel** [comment]
This can probably be closed now, because PR #34 has been merged and contains the relevant German translation data.

**Pull request closed without merging**